Source/Prism/Prism.csproj

```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;netcoreapp3.0;net45;uap10.0.15063;uap10.0.16299</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0;net45;uap10.0.15063;uap10.0.16299</TargetFrameworks>
```

Comment:
dropping netstandard1.0? Then line 19 to 21 can be removed, group starting line 23 can be merged with the group below it